src/Symfony/Component/Translation/Tests/Util/XliffUtilsTest.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+namespace Symfony\Component\Translation\Tests\Util;
+
+use Symfony\Component\Translation\Util\XliffUtils;
```

Comment:
The config component is currently only the require-dev https://github.com/symfony/symfony/blob/2.7/src/Symfony/Component/Translation/composer.json#L23

I would actually like this class to move its own component :)